### PR TITLE
Integration specs

### PR DESCRIPTION
**Pull request by awaisanwar544**
Summary:
Added integration tests with Capybara to:

User login page
User index page
User show page
Post index page
Post show page

Screenshot 2022-06-17 at 1 13 45 AM

Screenshot 2022-06-17 at 1 14 02 AM

### CODE REVIEW

File: spec/system/user_index_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    @user = User.create(id: 1,
+                        name: 'Awais',
+                        photo: 'https://via.placeholder.com/150',
+                        bio: 'Programmer',
+                        postsCounter: 0,
+                        email: 'test@test.com',
+                        password: 'password')
+  end
+
+  after(:all) do
```
Comment (BenMukebo) [inline spec/system/user_index_page_spec.rb:14]: - [x] Great job so far 👏🏻  However most of your tests are not passing. Kindly fix them:

newtest

**Review by BenMukebo — APPROVED** [review]
### Hi Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>



## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Kindly make sure you write instructions for running your tests in your readme

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/system/user_index_page_spec.rb (hunk 1 above)
Comment (awaisanwar544) [reply spec/system/user_index_page_spec.rb:14]: There must be some configuration issues on your machine or maybe you have the older version of ruby or rails. All the tests are running and passed on my system.

kindly have a look at this video

https://user-images.githubusercontent.com/87834222/174189174-d1400be4-7ce3-4623-bd42-07864386abf8.mov

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @awaisanwar544 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous reviewer
- Great communication with the reviewer

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/system/user_index_page_spec.rb (hunk 1 above)
Comment (BenMukebo) [reply spec/system/user_index_page_spec.rb:14]: ### I think your project is okay 👍🏻

**Pull request merged**